azurecontainerapps.ts

```diff
@@ -608,8 +608,8 @@ export class azurecontainerapps {
         } else if (this.shouldCreateOrUpdateContainerAppWithUp) {
             await this.appHelper.createOrUpdateContainerAppWithUp(this.containerAppName, this.resourceGroup, this.commandLineArgs);
         } else {
-            // Update the Container App using the 'up' command
-            await this.appHelper.updateContainerAppWithUp(this.containerAppName, this.resourceGroup, this.commandLineArgs, this.ingress, this.targetPort);
+            // Update the Container App using the 'containerapp update' and 'ingress update' commands
+            await this.appHelper.updateContainerAppWithUpdateAndIngress(this.containerAppName, this.resourceGroup, this.commandLineArgs, this.ingress, this.targetPort);
```

Comment:
I would be in favor of us having two separate calls here: one that updates the Container App and another that updates the ingress for the Container App

```typescript
// Update the Container App using the 'containerapp update' command
await this.appHelper.updateContainerAppWithUpdate(this.containerAppName, this.resourceGroup, this.commandLineArgs);

// Update the ingress and target port on the Container App using the 'ingress update' command
await this.appHelper.updateContainerAppIngress(this.containerAppName, this.resourceGroup, this.ingress, this.targetPort);
```

This will allow the functions to not be overloaded with functionality and be reusable in other scenarios we may want to use them in.

Reply:
makes sense! changing this in the next commit